src/main/java/cz/codecamp/logger/loggers/FileLogger.java

```diff
@@ -0,0 +1,34 @@
+package cz.codecamp.logger.loggers;
+
+import cz.codecamp.logger.LogLevelEnum;
+import cz.codecamp.logger.LoggerInterface;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+
+/**
+ * Created by filip on 03/10/16.
+ */
+public class FileLogger extends AbstractLogger {
+    @Override
+    public void log(LogLevelEnum level, String message) {
+        try {
+            PrintStream fileStream = new PrintStream(new FileOutputStream(new File("application.log"), true));
+            fileStream.println(level.toString() + message);
```

Comment:
- Otevírat soubor znovu a znovu s každou zalogovanou zprávou je hodně nehospodárné, na produkci ti to zabije server.
- Pokud už je stream otevřen v téhle metodě, musí tu být i zavřen.